Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -100,9 +100,9 @@ public void setupShooterBeltCommands(
             ShooterBeltsManagerSubsystem shooterBeltsSubsystem)
     {
         RunShooterBeltPowerCommand runBeltLeft = new RunShooterBeltPowerCommand(shooterBeltsSubsystem.getLeftBelt());
-        oi.leftButtons.getifAvailable(2).whileHeld(runBeltLeft);
+        oi.controller.getXboxButton(XboxButton.LeftStick).whileHeld(runBeltLeft);
```

Comment:
So to fire the shooter someone will hold down on a given side: Stick, Bumper and Trigger?
I'm wondering if we can simplify that down a bit now that we have a mostly functioning robot, for instance can we have 1 button that spins up the shooter and runs the belt while at speed?